lib/membrane_rtc_engine/endpoints/hls_endpoint.ex

```diff
@@ -17,6 +17,14 @@ if Enum.all?(
     ]
     ```
 
+    It can perform transcoding, in such case these plugins are also needed:
```

Comment:
I think we should provide more info about transcoding i.e. what it does and when someone might want to use it. The question is where this should be done. We could mention about transcoding in moduledoc and provide detailed description in `transcoding_config` option

Reply:
Fixed, details addded to `TranscodingConfig` moduledoc, which is mentioned in `hls_endpoint` moduledoc.